website/content/docs/configuration/server_join.mdx

```diff
@@ -225,3 +260,33 @@ project which have the given `tag_value`.
 Discovery requires a [GCE Service
 Account](https://cloud.google.com/compute/docs/access/service-accounts).
 Credentials are searched using the following paths, in order of precedence.
+
+
+### `go-netaddrs` Configurations
+
+`go-netaddrs` configurations are prefixed with `exec=`.
+
+What follows the `exec=` prefix may be any executable program and its arguments.
+Commands run by `go-netaddrs` must return a list of space-delimited IPv4 or IPv6
+addresses and exit with code `0` on success and non-zero on failure.
+
+Executables may be referenced by absolute file system path or by any program
+name discoverable by `os/exec`'s [LookPath](https://pkg.go.dev/os/exec#LookPath)
```

Comment:
I have a feeling this may be too Go-specific for the audience. While `LookPath` is a nice system-independent way to talk about program path discovery, it's rooted in Go, which Nomad users may not be familiar with. 

Any thoughts on a system-independent way to discuss path discovery without being go-specific?

Reply:
That's something worth considering, for sure (we run into this with our `template` docs and it's unfortunate). Here I think we can get away with a narrow description of the behavior and then refer to the Go docs:

> You can refer to the executable by its absolute file system path, or by the executable's name if it can be found on the Nomad agent's `PATH`. See [executables in the current directory](https://pkg.go.dev/os/exec#hdr-Executables_in_the_current_directory) for more details on the lookup behavior.